src/Libraries/CoreNodes/DateTime.cs

```diff
@@ -254,11 +265,11 @@ public enum DayOfWeek
     public static class TimeSpan
     {
         /// <summary>
-        ///     Yields a new TimeSpan calculated from the time difference between two DateTimes.
+        ///     Yields a new TimeSpan calculated from the time difference between two DateTimes
         /// </summary>
-        /// <param name="date1">Starting DateTime.</param>
-        /// <param name="date2">Ending DateTime.</param>
-        /// <returns name="timeSpan">TimeSpan</returns>
+        /// <param name="date1">Starting DateTime</param>
+        /// <param name="date2">Ending DateTime</param>
```

Comment:
I think the description of the arguments might be misleading, because in the standard use case `date1` would be later than `date2`. Otherwise the result would be a negative time span.

Reply:
Addressed in 06ff62e